op-guide/gc.md

```diff
@@ -0,0 +1,77 @@
+---
+title: TiDB 垃圾回收 (GC)
+category: advanced
+---
+
+# TiDB 垃圾回收 (GC)
+
+TiDB 采用 MVCC 的方式来进行并发控制。当对数据进行更新或者删除时，原有的数据不会被立刻删除，而是会被保留一段时间，并且在这段时间内这些旧数据仍然可以被读取。这使得写入操作和读取操作不必互斥，并使读取历史数据成为可能。
+
+存在超过一定时间并且不再使用的版本会被清理，否则它们将始终占用硬盘空间，并对性能产生负面影响。我们使用一个垃圾回收 (GC) 机制来清理这些旧数据。
+
+## 工作方式
+
+TiDB 会周期性地进行 GC 。每个 TiDB Server 启动后都会在后台运行一个 gc_worker ，每个集群中会有其中一个 gc_worker 被选为 leader ， leader 负责维护 GC 的状态并向所有的 TiKV region leader 发送 GC 命令。
```

Comment:
- Please delete the extra whitespaces between English characters and Chinese punctuation:
    - After "GC"
    - After "gc_worker"
    - After the 1st "leader"
    - Before the 2nd "leader"
- "region" -> "Region"